paper/src/main/java/com/denizenscript/denizen/paper/events/WorldGameRuleChangeScriptEvent.java

```diff
@@ -0,0 +1,144 @@
+package com.denizenscript.denizen.paper.events;
+
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.objects.WorldTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import io.papermc.paper.event.world.WorldGameRuleChangeEvent;
+import org.bukkit.command.BlockCommandSender;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.minecart.CommandMinecart;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+
+public class WorldGameRuleChangeScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // gamerule changes (in <world>)
+    //
+    // @Plugin Paper
+    //
+    // @Group Paper
+    //
+    // @Triggers when a gamerule changes.
+    //
+    // @Context
+    // <context.gamerule> returns name of the GameRule which was changed.

```

Comment:
`the` name of

Reply:
also, should have4 a switch for the gamerule probably

Reply:
Added a `gamerule:<gamerule>` switch and improved Meta.